app/views/posts/index.html.erb

```diff
@@ -1,6 +1,6 @@
-  <div class="posts">
+ <div class="posts">
     <%= render partial: "shared/user", locals: { user: @user } %>
-    <% @posts.each do |post| %>
+    <% @posts.includes(:comments).each do |post| %>
```

Comment:
- [ ] **[OPTIONAL]** Great job working on the previous reviewer's suggestions! 🚀 However, I would suggest that you move this `:includes` to your controller as it is meant to be part of the query. Applying this change will make your code more readable and better organized by following the Single Responsibility principle in which views are just responsible for displaying the data provided by the controller! 🤓 🔧 


```rb
# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  ...
  def index
    @user = User.find(params[:user_id])
    @posts = @user.posts.includes(:comments).order(created_at: :asc)
  end
  ...
end
```